src/main/java/org/broadinstitute/hellbender/tools/exome/germlinehmm/CopyNumberTriStateTransitionProbabilityCache.java

```diff
@@ -137,8 +137,16 @@ public double visit(final int to, final int from, final double probability) {
 
     /**
      * Wrapper for a {@link RealMatrix} that is indexed by the CopyNumberTriState enum
+     *
+     * @implNote This class is made public for Serialization issues. Otherwise, we get the following exception
+     * when running on GCS:
+     *
+     * java.lang.IllegalAccessException: Class com.twitter.chill.Instantiators$$anonfun$normalJava$1
+     * can not access a member of class org.broadinstitute.hellbender.tools.exome.germlinehmm.CopyNumberTriStateTransitionProbabilityCache
+     * $LogTransitionProbabilityMatrix with modifiers "public"
+     *
      */
-    private static class LogTransitionProbabilityMatrix implements Serializable {
+    public static class LogTransitionProbabilityMatrix implements Serializable {
```

Comment:
Not in this commit, but `final static int size` -> `private static final int NUMBER_OF_STATES` (or something along those lines) below.